website/public/js/services/taskServices.js

```diff
@@ -15,6 +15,30 @@
 
   function tasksFactory($rootScope, Shared, User) {
 
+    function addTask(listDef, fn) {
+      if (listDef.bulk) {
+        var tasks = listDef.newTask.split(/[\n\r]+/);
+        tasks.reverse();
+        _.each(tasks, function(task) {
+          fn(task);
+        });
+        listDef.bulk = false;
+      } else {
+        var task = listDef.newTask;
+        fn(task);
+      }
+      delete listDef.newTask;
+      delete listDef.focus;
+    }
+
+    function toggleBulk(list) {
+      if (typeof list.bulk === 'undefined') {
```

Comment:
I'm sure this doesn't matter but I don't think that's true.  I mean `typeof list.bulk === 'undefined'` is only true if `list.bulk` is undef. But `!list.bulk` would be true if it were 0,null, '', false, undefined and maybe others.  I'm probably misunderstanding you though.